G1ANT.Addon.Selenium/Commands/SeleniumOpenCommand.cs

```diff
@@ -52,6 +56,10 @@ public void Execute(Arguments arguments)
             try
             {
                 int chromePort = 0;
+                var chromeProfiles = new Dictionary<string, bool>();
+                if (arguments.ChromeProfiles != null)
+                    foreach (var pair in arguments.ChromeProfiles.Value)
+                        chromeProfiles.Add(pair.Key, (bool)pair.Value);
```

Comment:
Value of DictionaryStructure can be any C# type, use Convert.ToBoolean to convert to bool or throw an exception if the value is not bool type